### DISCUSSION TITLE
Teleport with Ansible with GitHub Actions

### DISCUSSION THREAD

**Discussion by simaqsous** [gravitational/teleport#45640] [Q&A]
I have the following issue/question:

I have followed this documentation 
https://goteleport.com/docs/enroll-resources/server-access/guides/ansible/
it works locally 
but when I try to implement it with GitHub Actions, here where the issue begins 
1. in gh actions I am authenticated teleport using token and bot
which are working, since I can list the nodes, tsh status and tsh config 
2. I still follow the above documentation but keep getting this error 
"No such device or address, Host verification failed"

any help would be highly appreciated

**Comment by KrisF-Midnight** [comment]
@simaqsous did you get to the bottom of this? I'm facing similar issues.
